tests/planner_test/tpch.planner.sql

```diff
@@ -146,21 +193,33 @@ order by
 limit 10;
 
 /*
-Projection: [l_orderkey, sum((l_extendedprice * (1 - l_discount))), o_orderdate, o_shippriority] (cost=63701.88)
-  TopN: limit=10, offset=0, orderby=[sum((l_extendedprice * (1 - l_discount))) desc, o_orderdate asc] (cost=63647.88)
-    Aggregate: [sum((l_extendedprice * (1 - l_discount)))], groupby=[l_orderkey, o_orderdate, o_shippriority] (cost=62224.105)
-      Projection: [o_orderdate, o_shippriority, l_orderkey, l_extendedprice, l_discount] (cost=59744.105)
-        HashJoin: inner, on=([o_orderkey] = [l_orderkey]) (cost=55264.105)
-          Projection: [o_orderkey, o_orderdate, o_shippriority] (cost=24811.05)
-            HashJoin: inner, on=([c_custkey] = [o_custkey]) (cost=22091.05)
-              Projection: [c_custkey] (cost=2940)
-                Filter: (c_mktsegment = 'BUILDING') (cost=2700)
-                  Scan: customer[c_custkey, c_mktsegment] (cost=2000)
-              Filter: (1995-03-15 > o_orderdate) (cost=7500)
-                Scan: orders[o_orderkey, o_custkey, o_orderdate, o_shippriority] (cost=4000)
-          Projection: [l_orderkey, l_extendedprice, l_discount] (cost=10220)
-            Filter: (l_shipdate > 1995-03-15) (cost=7500)
-              Scan: lineitem[l_orderkey, l_extendedprice, l_discount, l_shipdate] (cost=4000)
+
+Projection { cost: 63701.88, exprs: [ l_orderkey, sum {  }, o_orderdate, o_shippriority ] }                         
+└── TopN { cost: 63647.88, limit: 10, offset: 0, order_by: [ desc {  }, asc {  } ] }                                
+    └── Aggregate { aggs: [ sum {  } ], cost: 62224.105, group_by: [ l_orderkey, o_orderdate, o_shippriority ] }    
+        └── Projection                                                                                              
+            ├── cost: 59744.105                                                                                     
+            ├── exprs: [ o_orderdate, o_shippriority, l_orderkey, l_extendedprice, l_discount ]                     
+            └── HashJoin { cost: 55264.105, on: Equality { lhs: [ o_orderkey ], rhs: [ l_orderkey ] }, type: inner }
+                ├── Projection { cost: 24811.05, exprs: [ o_orderkey, o_orderdate, o_shippriority ] }               
+                │   └── HashJoin                                                                                    
+                │       ├── cost: 22091.05                                                                          
+                │       ├── on: Equality { lhs: [ c_custkey ], rhs: [ o_custkey ] }                                 
```

Comment:
why not?
```suggestion
                │       ├── on: = { lhs: [ c_custkey ], rhs: [ o_custkey ] }                                 
```

Reply:
Could you please propose the change to the generator? I can commit directly with your name if you do so

Reply:
https://github.com/risinglightdb/risinglight/pull/771/files#r1089681910